lint-rules-android-lint/src/main/java/com/vanniktech/lintrules/android/MatchingIdFixer.kt

```diff
@@ -0,0 +1,18 @@
+package com.vanniktech.lintrules.android
+
+import com.android.tools.lint.detector.api.XmlContext
+
+class MatchingIdFixer(context: XmlContext, private val id: String) {
+  private val layoutName = context.file.name.replace(".xml", "")
```

Comment:
Moved `layoutName` here. This will potentially would allow us to analyze it better. E.g split with `_` and check words and wrong order.